src/main/java/InputBoundary/playSpaceInputBoundary.java

```diff
@@ -12,7 +12,7 @@ public interface playSpaceInputBoundary {
      * function to play the space
      * @param songInputData data required to play the space
      */
-    default void playSpace(songInputData songInputData){}
+    default void playSpace(playSpaceInputData songInputData){}
```

Comment:
does it matter?

Reply:
Technically, yes, because you could have some weird implementation of the interface that doesn't implement the methods and it would not raise any errors. Unlikely, but just introducing functionality we didn't mean to

Reply:
ok sure